e2e/create-cli-e2e/tests/init.e2e.test.ts

```diff
@@ -48,8 +49,9 @@ describe('create-cli-node', () => {
     );
   });
 
-  it('should execute package correctly over npm init', async () => {
-    const cwd = join(baseDir, 'init');
+  // eslint-disable-next-line vitest/no-disabled-tests
+  it.skip('should execute package correctly over npm init', async () => {
```

Comment:
Could you disable the `node ...` test in favour of leaving this `npm init` test enabled? This test is closest to how a real user would execute it.

Reply:
It was flay in past runs, but sems stable now.